docs/src/content/docs/es/components/card-grids.mdx

```diff
@@ -0,0 +1,179 @@
+---
+title: Cuadrículas de tarjetas
+description: Aprende a envolver varias tarjetas en una cuadrícula en Starlight.
+sidebar:
+  order: 4
+---
+
+import { Card, CardGrid, LinkCard } from '@astrojs/starlight/components';
+
+Para envolver múltiples componentes [`<Card>`](/es/components/cards/) o [`<LinkCard>`](/es/components/link-cards/) en una cuadrícula, usa el componente `<CardGrid>`.
+
+import Preview from '~/components/component-preview.astro';
+
+<Preview>
+
+<CardGrid slot="preview">
+	<Card title="Stars" icon="star">
```

Comment:
```suggestion
	<Card title="Estrellas" icon="star">
```

I guess we can use the previous translation from the old component documentation?